### PR TITLE
RefactorKata -Hollis Medley

### PR DESCRIPTION
**Pull request by hmedley**
sorry for the long commits.

### CODE REVIEW

File: RefactorKata/RefactorKata/Program.cs [hunk 1]
```diff
@@ -1,41 +1,46 @@
...
             }
-            Conn.Dispose();
-            Console.WriteLine("Products Loaded!");
-            for (int i =0; i< products.Count; i++)
+        }
+
+        private static IEnumerable<Product> GetProducts()
+        {
+            using (var conn =
+                new SqlConnection("Server=.;Database=myDataBase;User Id=myUsername;Password = myPassword;"))
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: Let's extract the sql connection string to the app.config file and use ConfigurationManager.ConnectionStrings

File: RefactorKata/RefactorKata/Program.cs [hunk 2]
```diff
@@ -1,41 +1,46 @@
...
+
             {
-                Console.WriteLine(products[i].name);
+                var cmd = conn.CreateCommand();
+                cmd.CommandText = "select * from Products";
+                /*
+                 * cmd.CommandText = "Select * from Invoices";
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: Let's remove commented code that doesn't have an explaination

File: RefactorKata/RefactorKata/Program.cs (hunk 1 above)
Comment (hmedley) [reply RefactorKata/RefactorKata/Program.cs]: happy to comply but the project doesn't not contain a project config file. (Cody told me to say that)

Comment (soitgoes) [reply RefactorKata/RefactorKata/Program.cs]: We're reviewing tomorrow.

**Pull request closed without merging**